src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator("Excavator");
+        Bulldozer bulldozer = new Bulldozer("Bulldozer");
+        Truck truck = new Truck("Truck");
```

Comment:
Look at the [checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop)